department-of-veterans-affairs/vets-website
# Update current spouse former marriages

## Summary
Update disclosure titles on the Pension Benefits form (21P-527EZ)

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Continue to the current spouse's former marriages page of the 'Household information' chapter
3. Verify that the text fields have been updated to the web component (Other, Location of Marriage, Location of Separation)
4. Verify that the form data is able to be reviewed and edited on the 'Review application' page
5. Verify that the form data is able to be saved and submitted

## Acceptance criteria
[Issue in Github - 74871](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76775)

## Related issue(s)
n/a

## Testing done

## Screenshots
| Before | After |
| ------ | ------ |
|![localhost_3001_pension_application_527EZ_form-saved (3)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/9296c170-25f7-4a6a-8213-053944925f55)|![localhost_3001_pension_application_527EZ_form-saved (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/592b6dee-10d8-4f59-a935-5ee625dd858b)



## What areas of the site does it impact?
Pension Benefits Application

### Quality Assurance & Testing

### Error Handling


### Authentication
